src/components/ToggleSwitch/ToggleSwitch.tsx

```diff
@@ -44,16 +44,18 @@ export const ToggleSwitch: FC<ToggleSwitchProps> = ({
 }) => {
   const id = useId();
   const theme = mergeDeep(useTheme().theme.toggleSwitch, customTheme);
+  const buttonRef = useRef<HTMLButtonElement>(null);
```

Comment:
isn't this redundant not being used anywhere? 🤔

Reply:
Yes. Dead code. Thanks for noticing.